.github/workflows/osrm-backend.yml

```diff
@@ -138,7 +138,7 @@ jobs:
             runs-on: ubuntu-20.04
             BUILD_TOOLS: ON
             BUILD_TYPE: Debug
-            CLANG_VERSION: 6.0.0
+            CLANG_VERSION: 6
```

Comment:
We could just use `CCOMPILER/CXXCOMPILER` to use pre-installed clang, but the lowest pre-installed version is only 10: https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md